.github/workflows/build-test-docker.yaml

```diff
@@ -46,6 +50,10 @@ on:
         required: true
         type: string
         description: Dockerfile to build
+      target:
```

Comment:
There is no way to factorize things with the workflow_dispatch above to avoid repeating ourselves?

Reply:
I looked around and unfortunately it doesn't seem like there's any way to template this :(